sql/migrations/20211203202918_world.sql

```diff
@@ -0,0 +1,72 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20211203202918');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20211203202918');
+-- Add your query below.
+
+
+-- Add Scripts to Quest Stones of Binding (credit cmangos)
+INSERT INTO `gameobject_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(15216, 0, 9, 443, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of East Binding - Respawn Gameobject'),
+(16598, 0, 9, 444, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of Outer Binding - Respawn Gameobject'),
+(15398, 0, 9, 445, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of West Binding - Respawn Gameobject');

```

Comment:
I am confused whats going on here. Why are you respawning Incendicite Mineral Veins upon using this object?